ml-lifecycle/feature_store/FS_demo.ipynb

```diff
@@ -24,7 +24,6 @@
    "source": [
```

Comment:
<p><code>Feature Store</code> rather than <code>FeatureStore</code>.</p><p> </p><p>You could also remove the <code>new</code> in "new SageMaker capability" since it was released 7+ months ago now.</p><p> </p><p>Typo - change <code>erves</code>  to <code>serves</code>  .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2814/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>